sdk/attestation/azure-security-attestation/pom.xml

```diff
@@ -115,6 +114,21 @@
               </argLine>
             </configuration>
           </plugin>
+
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-enforcer-plugin;external_dependency} -->
+            <configuration>
+              <rules>
+                <bannedDependencies>
+                  <includes>
+                    <include>com.nimbusds:nimbus-jose-jwt:[9.10]</include> <!-- {x-include-update;com.nimbusds:nimbus-jose-jwt;external_dependency} -->
+                  </includes>
+                </bannedDependencies>
+              </rules>
+            </configuration>
+          </plugin>
```

Comment:
This should be defined in the default profile. Currently, it's defined in `java-lts` profile (line 98) which works only for Java 11 or above.

Reply:
Ah, makes sense.